src/pass_manager.cpp

```diff
@@ -101,7 +102,16 @@ struct module_pm : module_pass_manager
         trace("Module: ", mod->name(), ", Pass: ", p.name());
         const double t1 = ts.record<seconds>();
         assert(mod->validate() == mod->end());
-        p.apply(*this);
+        if(enabled(MIGRAPHX_TIME_PASSES{}))
```

Comment:
we probably want to consolidate this and update the previous timer code.

Reply:
I see, I will remove the extra timer code.